sdk/src/auth/types.ts

```diff
@@ -1,5 +1,7 @@
 export type AuthenticationMode = 'json' | 'cookie' | 'session';
 
+export type LoginPayload = { email: string; password: string } | { identifier: string; password: string };
```

Comment:
May be worth improving this type def as it now allows all 3 to be passed instead of email OR identifier.